src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Data/EditBranchVersioning/EditBranchVersioning.xaml.cs

```diff
@@ -205,7 +205,7 @@ private async void MyMapView_GeoViewTapped(object sender, GeoViewInputEventArgs
                     await _selectedFeature.FeatureTable.UpdateFeatureAsync(_selectedFeature);
```

Comment:
On edit with branch versioning, the edited feature disappears after editing. It reappears when switching back and forth between versions.

Reply:
I tested the sample again be reverting the changes and this issue doesn't happen. I'm not sure if this sample is suppose to be changed with this issue but I'll reach out to Shubham and Mike to get more info.